schema/data/deploy-data.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+set -e
+
+database=${PGDATABASE:-ggircs_dev}
+
+# =============================================================================
+# Usage:
+# -----------------------------------------------------------------------------
+usage() {
+    cat << EOF
+$0 [-d] [-p] [-s] [-h]
+
+Upserts test data in the $database database, and deploys the schemas using sqitch if needed.
+If run without the corresponding options, this script will deploy the swrs and portal schemas
+if they do not exist, and then insert the portal test data.
+
+Options
+
+  -d, --drop-db
+    Drops the $database database before deploying
+  -s, --deploy-swrs-schema
+    Redeploys the swrs schema and inserts the swrs test reports. This requires the .cas-ggircs submodule to be initialized
+  -p, --deploy-portal-schema
+    Redeploys the portal schema
+  -h, --help
+    Prints this message
+
+EOF
```

Comment:
Add the documentation for the new options

Reply:
Added documentation for new flags